src/main/java/mate/academy/controller/AuthenticationController.java

```diff
@@ -27,4 +37,17 @@ public UserResponseDto register(@RequestBody @Valid UserRegistrationDto userRequ
                 userRequestDto.getPassword());
         return userMapper.mapToDto(user);
     }
+
+    @PostMapping("/login")
+    public ResponseEntity<Object> login(@RequestBody @Valid UserLoginDto userLoginDto)
+            throws AuthenticationException {
+        User user = authenticationService.login(userLoginDto.getLogin(),
+                userLoginDto.getPassword());
+        String token = jwtTokenProvider.createToken(user.getEmail(),
+                user.getRoles()
+                        .stream()
+                        .map(role -> role.getRoleName().name())
+                        .collect(Collectors.toList()));
```

Comment:
I would move this part to some service class, because there no business logic should be present on the controller later